testdata/testoutput5

```diff
@@ -5312,6 +5312,259 @@ No match
 
 # End caseless restrict tests
 
+# TESTS for PCRE2_EXTRA_TURKISH_CASING - again, tests with and without.
+
+/i/i,utf
+    i
+ 0: i
+    I
+ 0: I
+    \x{0130}
+No match
+    \x{0131}
+No match
+
+/i/i,utf,turkish_casing
+    i
+ 0: i
+    I
+No match
+    \x{0130}
+ 0: \x{130}
+    \x{0131}
+No match
+
+/I/i,utf
+    i
+ 0: i
+    I
+ 0: I
+    \x{0130}
+No match
+    \x{0131}
+No match
+
+/I/i,utf,turkish_casing
+    i
+No match
+    I
+ 0: I
+    \x{0130}
+No match
+    \x{0131}
+ 0: \x{131}
+
+/\x{0130}/i,utf
+    i
+No match
```

Comment:
Usually is better to sort them and make the ones expected to match at the end after something like "\= Expect no match", so it is harder to miss  bug like this one.

Reply:
Thanks, good point.